process_blocking.go

```diff
@@ -184,10 +184,10 @@ func (p *blockingProcess) Info(ctx context.Context) ProcessInfo {
 		case res := <-out:
 			return res
 		case <-ctx.Done():
-			return ProcessInfo{}
+			return p.getInfo()
```

Comment:
It's worth noting that `getInfo()` will return an empty `ProcessInfo` if the process has not yet terminated and the context has reached its deadline or was canceled. This is actually not as good as we _could_ be. A better idea here is probably to change the `blockingProcess` implementation so that it holds a `cmd` object, and uses this `cmd` object to return a partially-filled-in `ProcessInfo` struct from `getInfo()`.